<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistortionApplicationParametersControlWidget</class> 
 <widget class="QWidget" name="DistortionApplicationParametersControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DistortionApplicationParameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>Force Scale</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="forceScaleCheckBox">
       <property name="toolTip">
         <string>Force Scale</string>
       </property>
       <property name="checked">
           <bool>0</bool>
       </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>Adopt Scale</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="adoptScaleCheckBox">
       <property name="toolTip">
         <string>Adopt Scale</string>
       </property>
       <property name="checked">
           <bool>0</bool>
       </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>Resize policy</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="resizePolicyComboBox">
       <property name="toolTip">
         <string>Resize policy</string>
       </property>
       <property name="currentIndex">
         <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>No Change</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Force Size</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>To Fit Result</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>To No Gaps</string>
        </property>
       </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
       <property name="text">
         <string>New H</string>
       </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="newHSpinBox">
       <property name="toolTip">
         <string>New H</string>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
       <property name="text">
         <string>New W</string>
       </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="newWSpinBox">
       <property name="toolTip">
         <string>New W</string>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>forceScaleCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>DistortionApplicationParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>adoptScaleCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>DistortionApplicationParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>resizePolicyComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DistortionApplicationParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>newHSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DistortionApplicationParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>newWSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DistortionApplicationParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
 </customwidgets>
</ui>
